CRM/Prospect/Upgrader/Steps/Step1007.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+/**
+ * Class CRM_Prospect_Upgrader_Steps_Step1007.
+ */
+class CRM_Prospect_Upgrader_Steps_Step1007 {
+
+  /**
+   * Removes the 'Campaign_Id' Custom Field
+   *
+   * @return bool
+   *   Return value in boolean.
+   */
+  public function apply() {
+    $result = civicrm_api3('CustomField', 'getsingle', [
+      'return' => ["id"],
+      'custom_group_id' => "Prospect_Financial_Information",
```

Comment:
Let's use single quotes for array values  here e.g `Prospect_Financial_Information`
Also `getsingle` will throw an exception if not found, so it might be good to wrap it in a try catch block.